frontend/locales/de/translation.json

```diff
@@ -222,7 +222,8 @@
       "to public": "Öffentlich machen",
       "to private": "Privat machen",
       "to shared with admin": "Mit Administratoren teilen",
-      "to shared with everyone": "Allen Benutzern automatisch anzeigen"
+      "to shared with everyone": "Allen Benutzern automatisch anzeigen",
+      "to shared with everyone explanation": "Die Spur wird allen Benutzern automatisch und schreibgeschützt angezeigt. Andere Benutzer können also keine Änderungen vornehmen."
```

Comment:
This line is in a different place than in the english translation file and therefore doesn't get shown. Apparently it should be under "timeline.access" (and without the "to").